pkg/csi/cinder/mount/mount.go

```diff
@@ -121,6 +121,7 @@ func (m *Mount) GetDevicePath(volumeID string) (string, error) {
 		if devicePath != "" {
 			return true, nil
 		}
+		probeVolume()
```

Comment:
can you insert comment to this row `// see issue https://github.com/kubernetes/cloud-provider-openstack/issues/705`

Reply:
I added the comment